GUI/QtQuick/qml/Send/SendPopup.qml

```diff
@@ -112,7 +112,7 @@ CustomPopup {
     Connections {
         target: bsApp
         onTransactionExported: (text) => {
-            successDialog.details_text = text
+            successDialog.details_text = qsTr("Transaction successfully exported to ") + text
```

Comment:
This should be a format with arg placeholder: `qsTr("... to %1").arg(text)` as in other languages the position of the pathname can be in the middle of the sentence.